D8-launch-tasks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+read -p "Enter the site shortname on ACSF (e.g., \"virtualvisit\", no quotes): " sitename
+# Baby, what time is it?
+timestamp=$(date +%Y%m%d%H%M%S)
+# Generate a random 18-character password that can include mixed-case,
+# alphanumeric, and "_-!.+".
+jamiepass=$(cat /dev/urandom | LC_ALL=C tr -cd a-zA-Z0-9\_\-\!\.\+ | fold -w18 | head -n 1)
```

Comment:
I get this error with this: `tr: range-endpoints of '_-!' are in reverse collating sequence order`

I ran it in AFS. 

![Screen Shot 2021-06-02 at 3 07 12 PM](https://user-images.githubusercontent.com/8387457/120552229-370c1380-c3b4-11eb-86cc-5234305c991d.png)

https://stackoverflow.com/questions/18414353/error-about-the-tr-command-of-linux-shell

Reply:
Also useless cat. 

https://github.com/koalaman/shellcheck/wiki/SC2002